### DISCUSSION TITLE
Table not loading all the rows

### DISCUSSION THREAD

**Discussion by Ken-JIAnomads** [AntlerVC/firetable#1266] [Support / Q&A]
Hi,

I was adding a new row in Rowy with new data and it failed to load the new row (even though the new data/document shows up in Firebase).  Only shows 5 of 6 loaded (see attached). Any ideas?

![Screenshot 2023-05-22 at 8 08 07 pm](https://github.com/rowyio/rowy/assets/130031825/f170c518-6d08-404a-bb77-ab56739710ac)

**Comment by harinij** [comment]
Hi @Ken-JIAnomads - I see that you closed the issue - checking if this was resolved? if not just want to check if you have any sorting applied to any of the columns?

**Comment by Ken-JIAnomads** [comment]
Harini,

Thanks for checking in. I did have sorting applied to a column 'rank' and I
hadn't assigned a value in that particular new row of data.  Could that be
the reason it didn't load ?

I ended up adding remaining information directly in Firebase and it then
showed back up in Rowy.

Thanks
Ken

**Reply by harinij** [reply]
Yes that makes sense, as empty values when sorted will not be shown on the Table UI. We are working on a fix with better UI indication for this to enable/disable sorting so that will resolve this issue.

**Comment by Ken-JIAnomads** [comment]
Yes that would be very helpful!

Love the product btw, starting to bring  my team into using it. :)
